homeassistant/components/recorder/filters.py

```diff
@@ -153,30 +153,28 @@ def _generate_filter_for_columns(
 
         # Case 4 - both includes and excludes specified
         # Case 4a - include domain or glob specified
-        #  - if domain is included, pass if entity not excluded
-        #  - if glob is included, pass if entity and domain not excluded
-        #  - if domain and glob are not included, pass if entity is included
-        # note: if both include domain matches then exclude domains ignored.
-        #   If glob matches then exclude domains and glob checked
+        #  - if entity_id is included, accept
```

Comment:
let's not use accept/reject here

Reply:
I'm going to sync all the comments to the docs and make this case 1/2/3/4/5/6 instead of 1/2/3/4a/4b/4c